### PR TITLE
add sample docker test case configuration and refactor maven dependen…

### PR DESCRIPTION
**Pull request by bq-xiao**
Changed below:
+ add dubbo-samples-docker test case configuration file
+ refactor maven dependencies use dependencyManagement

### CODE REVIEW

File: dubbo-samples-docker/pom.xml [hunk 1]
```diff
@@ -42,19 +42,46 @@
...
             <artifactId>dubbo</artifactId>
-            <version>${dubbo.version}</version>
         </dependency>
 
         <dependency>
-            <groupId>org.apache.dubbo</groupId>
-            <artifactId>dubbo-dependencies-zookeeper</artifactId>
-            <version>${dubbo.version}</version>
-            <type>pom</type>
+            <groupId>org.apache.curator</groupId>
```
Comment (kylixs) [inline dubbo-samples-docker/pom.xml:78]: 原来的dubbo-dependencies-zookeeper是正确的，只需要删除version，因为上面管理了

File: dubbo-samples-docker/pom.xml [hunk 2]
```diff
@@ -42,19 +42,46 @@
...
+                <version>${dubbo.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.apache.dubbo</groupId>
+                <artifactId>dubbo-dependencies-zookeeper</artifactId>
+                <version>${dubbo.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
```
[the remark this answers is not in the document]
Comment (bq-xiao) [reply dubbo-samples-docker/pom.xml]: OK，已修改

File: dubbo-samples-docker/pom.xml [hunk 3]
```diff
@@ -89,23 +114,6 @@
             <id>dubbo-integration-test</id>
```
[the remark this answers is not in the document]
Comment (bq-xiao) [reply dubbo-samples-docker/pom.xml]: 已修改

File: dubbo-samples-docker/pom.xml (hunk 1 above)
Comment (bq-xiao) [reply dubbo-samples-docker/pom.xml]: 已修改

**Review by kylixs — APPROVED** [review]
It's great

File: dubbo-samples-docker/pom.xml [hunk 4]
```diff
@@ -42,18 +42,40 @@
         <zookeeper.port>2181</zookeeper.port>
```
Comment (kylixs) [inline dubbo-samples-docker/pom.xml:42]: 请删除pom中不需要的属性，如docker-maven-plugin.version/jib-maven-plugin.version/maven-failsafe-plugin.version/image.name/java-image.name/dubbo.port/zookeeper.port

Comment (bq-xiao) [reply dubbo-samples-docker/pom.xml:42]: OK，已删除未使用的properties

**Pull request merged**